src/main.cpp

```diff
@@ -264,4 +273,86 @@ void runSteeringDisplay() {
 	while (1) {
 		display.run();
 	}
+}
+
+void buttonRead(int buttonPress){
+	int ledPos;
+	for(int i = 0; i <8; i++){
+		shiftClk = 1;
+		wait_us(20);
+		shiftClk = 0;
+		shiftLatch = 1;
```

Comment:
I believe the latch only needs to go high at the start of the loop, not every loop. You can then set it low after the end of all the loops.

Reply:
Okay, got it!